src/extractor.rs

```diff
@@ -133,9 +137,14 @@ pub fn ensure_extracted() -> Result<(), ()> {
         if !crate::file_exists(table.target_file()) {
             let name = format!("-{}.sql.gz", table.name());
             let data_dir = std::fs::read_dir("data").unwrap();
-            let mut files = data_dir
-                .map(|entry| entry.unwrap().file_name().into_string().unwrap())
-                .filter(|fname| fname.ends_with(&name));
+            let mut files = data_dir.filter_map(|entry| {
+                let entry = entry.unwrap();
+                if entry.file_name().into_string().unwrap().ends_with(&name) {
+                    Some(entry.path())
+                } else {
+                    None
+                }
+            });
```

Comment:
Well, I'm not sure this is really more readable but I guess it makes clippy happy.